src/lib/libglut.js

```diff
@@ -633,10 +646,9 @@ var LibraryGLUT = {
   },
 
   glutMainLoop__proxy: 'sync',
-  glutMainLoop__deps: ['$GLUT', 'glutReshapeWindow', 'glutPostRedisplay'],
+  glutMainLoop__deps: ['$GLUT', 'glutPostRedisplay'],
   glutMainLoop: () => {
-    var canvas = Browser.getCanvas();
-    _glutReshapeWindow(canvas.width, canvas.height);
+    GLUT.onResize();
```

Comment:
Maybe add a comment here? It's not obvious to me why we need to resize when starting the main loop.

Reply:
Oh, right, thanks @sbc100 

Still, the new comment can't hurt, thanks @erik-larsen